<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SethostDialog</class>
 <widget class="QDialog" name="SethostDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络设置</string>
  </property>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>主机IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>端  口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius: 10px;
border: 2px groove purple;}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius: 10px;
border: 2px groove purple;}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bn_Connect">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接 &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/装逼背景2.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_ip</zorder>
  <zorder>label_port</zorder>
  <zorder>lineEdit_ip</zorder>
  <zorder>lineEdit_port</zorder>
  <zorder>Bn_Connect</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
